src/ui/components/VerifyPassword/VerifyPassword.tsx

```diff
@@ -75,9 +77,7 @@ const VerifyPassword = ({
       // TODO: @bao-sotatek handle error for this
     }
 
-    if (hint) {
-      setStoredHint(`${hint}`);
-    }
+    setStoredHint(hint ? String(hint) : "");
```

Comment:
Was there an issue with the code above?

In any case, I would recommend replacing this:

```
setStoredHint(hint ? String(hint) : "")
```

with this:

```
setStoredHint(`${hint || ""}`);
```

That's because template literals use less resources to get the same work done.

Reply:
@sdisalvo-crd 

Now we are displaying submit button of `VerifyPassword` Modal base on password hint. If we have hint submit button is `forgot password`, `reset my password` for another case.

If you have password hint and after that you changed new password with no hint, back to `ManagerPassword` page and open `VerifyPassword` modal and you can see submit button of `VerifyPassword` still `forgot password` not  `reset my password`.

Because this modal still save previous password hint in `storedHint` state. And when we get hint from storage, it now is empty but we only save it when password hint has value => display wrong submit button.